instrumentation/opentelemetry-instrumentation-aws-sdk/src/AwsSdkInstrumentation.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+use Opentelemetry\API\Common\Instrumentation\InstrumentationInterface;
+use Opentelemetry\API\Common\Instrumentation\InstrumentationTrait;
+
+
+class AwsSdkInstrumentation implements InstrumentationInteface
+{
+
+    use InstrumentationTrait;
+
+    private $instrumented;
+
+    public function getName(): string
+    {
+        return 'AWS SDK Instrumentation';
+    }
+
+    public function getVersion(): ?string
+    {
+        return '0.0.1';
+    }
+
+    public function getSchemaUrl(): ?string
+    {
+        return true;
+    }
+
+    public function init(): bool
+    {
+        return false;
+    }
+
+    function __construct()
+    {
+        $this->instrumented = false;
+    }
+
+    function activate()
```

Comment:
Can you please give this public access (same as the [interface](https://github.com/open-telemetry/opentelemetry-php/blob/main/src/API/Common/Instrumentation/InstrumentationInterface.php#L24))?